### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/attendance#1]
While I usually forgo detailed docs in favor of readable code, in this case I think users of the system would benefit from some help on the interface, and an as-non-techical-as-possible setup guide would be good.

**Comment by stenington** [comment]
@toolness or @christensenep either of you know what django's [admindocs](https://docs.djangoproject.com/en/1.7/ref/contrib/admin/admindocs/#module-django.contrib.admindocs) looks like?

**Comment by ghost** [comment]
I haven't used it myself, but it seem like it would be helpful.  It is oddly difficult to find any actual examples of what the end result looks like, though.

**Comment by toolness** [comment]
Yeah, I have busted up admindocs for the Hive directory, but I've found that they're really most useful for template designers and developers rather than for administrators of the system, IMO. Here are some screenshots from the admindocs of the [Hive Directory](https://github.com/toolness/hive-django) to give you an idea...

This is the front page of the admindocs:

![docs_frontpage](https://cloud.githubusercontent.com/assets/124687/5840387/14262100-a163-11e4-8d7e-64edb9611631.png)

If you click on the "tags" link, you get this:

![docs_templatetags](https://cloud.githubusercontent.com/assets/124687/5840391/1ef214fe-a163-11e4-8735-72515a95825e.png)

If you click on the "views" link, you get this:

![docs_view](https://cloud.githubusercontent.com/assets/124687/5840397/2d53328a-a163-11e4-9df4-f089cd543e70.png)

So yeah, it is pretty technical. Likely useful to onboarding new designers/developers, but not so much new administrators.

**Comment by stenington** [comment]
Ok, thanks! I think we need another more user-focused doc solution then.

**Comment by toolness** [comment]
Let me know what you decide on! I've been looking for a similar solution myself. Right now I basically just have a discourse thread where I post little guides and answer people's questions.

**Comment by stenington** [comment]
Hmm that's actually kind of nice idea if it might let users sort of "crowd-source" guides to help each other out.

**Comment by stenington** [comment]
Uh, but at the very least a starting point here would be good, if not a completely complete, addresses-everything-you-could-imagine sort of thing.

**Comment by toolness** [comment]
Yeah, I basically just made a new category in our discourse called "Hive Directory" and added a few pinned threads:
- A **Hive Directory News** pinned thread that is low-volume, where I just post incremental updates I've made to the software. This can be used in place of an announcement mailing list, because Discourse allows for per-thread email notifications--so users who just want to be notified of changes to the software can watch only this one thread.
- A **Hive Directory Admin FAQ** pinned thread where I constantly revamp the original post to reflect all the frequently-asked questions.

Aside from that, I also welcome people to ask questions and make suggestions etc. in the Hive Directory category itself. People who are super invested in the directory can then subscribe to email notifications for the whole category if they want.

**Comment by stenington** [comment]
@toolness I'm curious what kind of level of effort it takes to run a discourse instance?

**Comment by toolness** [comment]
If you bust it up with the ol' Docker, which they have [great instructions](https://github.com/discourse/discourse/blob/master/docs/INSTALL-digital-ocean.md) for, you can get it up and running in 30 minutes or less. And I believe that the deployment even self-updates itself, which is crazy cool... If not, it at least pokes you to upgrade whenever it detects that there's a new version, a la Wordpress, which is nice.

The big "cost" for DIscourse IMO isn't time but money... It wants a relatively beefy machine to run on: 1 GB RAM minimum, which on Digital Ocean costs $10/month. So it's not really a lightweight thing you can just throw up on a free Heroku instance.

**Comment by ghost** [comment]
I actually have some experience running Discourse (nothing that went to production, but I've run a staging version for an addon I was building for it), so if you want me to take point on that, I'd be happy to.  Assuming we feel like going that route, of course.

**Comment by toolness** [comment]
Discourse rocks! The only two downsides I can think of are:
- the beefiness of the system required to run it
- the super stingy defaults that prevent new users from doing lots of stuff for fear that they may be a spammer (these can be changed through the admin UI but there are a _lot_ of default settings and you're likely to miss a few until someone pipes up about how they couldn't do some really simple thing on the site).

Other than that it is the most awesome awesome thing ever. And the posts support Markdown and basic HTML so anyone who's familiar with GitHub will feel right at home!

**Comment by toolness** [comment]
However, I am also a forum junkie so my views may be biased.
